components/navbar/navbar.snip.html

```diff
@@ -77,9 +78,17 @@
           {{> ../calltracking/calltracking.snip.html}}
         {{/calltracking}}
       </div>
+      {{#state}}
```

Comment:
I tried that initially, but decided against it because I would need to do something like:

[class]="cart.added ? 'text-decoration-none mr2 ampstart-headerbar-fixed-link {{classes}} commerce-cart-added relative' : ' text-decoration-none mr2 ampstart-headerbar-fixed-link {{classes}}'

Which could be a difficult to read. Plus by having a wrapper it is more scalable and applies to the whole wrapper as opposed to just the link.